app/models/publisher_status_update.rb

```diff
@@ -0,0 +1,11 @@
+class PublisherStatusUpdate < ApplicationRecord
+  has_paper_trail
+  
+  ALL_STATUSES = ["created", "onboarding", "active"]
```

Comment:
Would you add the `suspended` state to this list?

Additionally, let's make these constants:

```
CREATED = 'created'
ONBOARDING = 'onboarding'
ACTIVE = 'active'
SUSPENDED = 'suspended'
ALL_STATUSES = [CREATED, ONBOARDING, ACTIVE, SUSPENDED]
``